en_US/extensions/plugins.md

```diff
@@ -179,30 +181,53 @@ This generates a new tarball, `my_emqx_plugin-1.0.0.tar.gz`, which you can uploa
 
 #### Write Configuration Schema for the Plugin (Optional)
 
+EMQX version 5.7.0 introduced REST API for plugin configuration management and Avro Schema for configuration validation, enhancing the ability to update plugin configurations dynamically during runtime.
+In addition, if you write an Avro Schema to validate the plugin's configuration, you are required to provide a default configuration file that matches the Avro Schema rules. This file should be located in `priv/config.hocon`.
```

Comment:
```suggestion
In addition, if you write an Avro Schema to validate the plugin's configuration, you are required to provide a default configuration file that matches the Avro Schema rules. This file should be saved as `priv/config.hocon`.
```

Reply:
This description explains the path that the default configuration file path should use.
"saved as" is more like a runtime operation.

Reply:
located in should be followed by a dir name.
maybe this then:
This file should be located in `priv` dir and named `config.hocon`.